simapp/app.go

```diff
@@ -655,9 +606,7 @@ func GetMaccPerms() map[string][]string {
 }
 
 // initParamsKeeper init params keeper and its subspaces
-func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino, key, tkey storetypes.StoreKey) paramskeeper.Keeper {
-	paramsKeeper := paramskeeper.NewKeeper(appCodec, legacyAmino, key, tkey)
-
+func initParamsKeeper(paramsKeeper paramskeeper.Keeper) {
```

Comment:
out of curiosity: How would this function look in the final app-wired simapp?

Reply:
it would be gone